README.md

```diff
@@ -71,29 +72,28 @@ The configuration passed into the resquel library is where the magic happens.
     {
       "method": "get",
       "endpoint": "/customer/:customerId",
-      "query": "SELECT * FROM customers WHERE id={{ params.customerId }};"
+      "query": "SELECT * FROM customers WHERE id={{ data.params.customerId }};"
     },
     {
       "method": "put",
       "endpoint": "/customer/:customerId",
-      "query": "UPDATE customers SET firstName='{{ data.firstName }}', lastName='{{ data.lastName }}', email='{{ data.email }}' WHERE id={{ params.customerId }};SELECT * FROM customers WHERE id={{ params.customerId }};"
+      "query": "UPDATE customers SET firstName='{{ data.firstName }}', lastName='{{ data.lastName }}', email='{{ data.email }}' WHERE id={{ data.params.customerId }};SELECT * FROM customers WHERE id={{ data.params.customerId }};"
```

Comment:
Why did you change this to ```data.params```? What was broken about just using ```params```?

Reply:
Well, using just params returned undefined for me. I checked the data in the queryReplace function and it contained the params object, so I started using data.params and it worked.